SPECS/node_exporter.spec

```diff
@@ -30,8 +30,11 @@ This package contains binary to export node metrics to prometheus.
 
 %install
 BREAKING_VERSION='0.15.0'
-#=0*10^2+15*10^1+0*10^0. See version_to_number()
-BREAKING_VERSION_INT='150'
+#=0*50^2+15*50^1+0*50^0. See version_to_number()
+BREAKING_VERSION_INT='750'
```

Comment:
looks like version_to_number() needs update. that function ends up evaluating 0.15.0 as version greater than 1.0.0 which is not true.

Reply:
i do have more robust version comparison in other scripts somewhere, so i'll find that and update it here.